src/NHibernate/Dialect/SQLiteDialect.cs

```diff
@@ -141,6 +141,7 @@ protected virtual void RegisterFunctions()
 			"longtext",
 			"memo",
 			"money",
+			"native",
```

Comment:
According to the test failure on AppVeyor, the missing word was `nothing`, not `native`.

Reply:
True. Does not work anyway.